.github/workflows/release_notes.yml

```
@@ -0,0 +1,41 @@
+name: Deploy Release Notes Function
+
+on:
+  push:
+    tags:
+      - "*"
```

It would be better to trigger this manually when needed. 
```suggestion
on: workflow_dispatch
```

---

Changed here: 2c5d73c1df6941b544cb751cbb6573969897514c